src/ape/pytest/runners.py

```diff
@@ -23,6 +23,7 @@ def __init__(
         self.receipt_capture = receipt_capture
         self._provider_is_connected = False
         ape.reverts = RevertsContextManager  # type: ignore
+        ape.default_sender = default_sender  # type: ignore
```

Comment:
Actually think this might be good in a general sense, not specific to pytest. just add to `ape/__init__.py`